src/test/ui/argument-suggestions/complex.stderr

```diff
@@ -19,8 +19,8 @@ LL | fn complex(_i: u32, _s: &str, _e: E, _f: F, _g: G, _x: X, _y: Y, _z: Z ) {}
    |    ^^^^^^^ -------  --------  -----  -----  -----  -----  -----  ------
 help: the following changes might help
    |
-LL |   complex( {u32},, &"", {E}, F::X2, G{}, X {}, Y {}, Z {});
-   |            ^^^^^^ --    ^^^^ ^^^^^  ^^^  ^^^^  ^^^^  ^^^^
+LL |   complex({u32}, &"", {E}, F::X2, G{}, X {}, Y {}, Z {});
+   |           ^^--^       ^^^^ ^^^^^  ^^^  ^^^^  ^^^^  ^^^^
```

Comment:
`^^--^` looks weird, why did that change?

Reply:
Yea, I'm not sure what caused this; I don't really understand the difference between ^^ and --?  maybe -- is removed text?

Reply:
^^^^ is a 'primary span' I think. I'm not 100% sure.

Reply:
Outside of suggesitons, `--` is secondary spans, that is showing two spans, one pointing at `{u32}` and another pointing at `32`. On suggestions, like here, `--` means something removed in between these. I think that there might be a rendering error if the multispan suggestions are not sorted :-/

Reply:
@estebank ooo, good to know! thanks!

Reply:
I sorted, and this is still happening.  I'm going to focus on other oddities for now.